StepsToCare/CareDashboard.ascx.cs

```diff
@@ -1777,6 +1820,20 @@ private void AddDynamicControls( bool addColumns = true )
             {
                 using ( var rockContext = new RockContext() )
                 {
+                    if ( TargetPerson != null )
```

Comment:
Have status column show when filter is set to ""

Reply:
@guymaness2 I ended up tweaking this a little more than anticipated, due to some issues with the order of operation and it affecting control id's on the grid. Let me know if you want to re-review it or if async it still makes sense https://github.com/KingdomFirst/RockBlocks/pull/103/commits/bde3daf25cb2d17a4ab45befb03f3c34c6329abc